scripts/funcotator/data_sources/gnomAD/createGnomadAlleleFreqTsv.wdl

```diff
@@ -0,0 +1,114 @@
+# Create a TSV containing genomic position, dbSNP ID, alleles, and the allele frequency from v2.1 of gnomAD (hg19/b37).
```

Comment:
Is this a supported script?

Reply:
Yes - I want to make it available so that anyone needing to manipulate gnomAD in the future can do so.